error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1232,6 +1232,19 @@ StepVerifier.Step<T> after(StepVerifier.Step<T> step, T object) {
     }
   }
 
+  /** Remove redundant {@code StepVerifier.Step#expectNextCount(0)} checks. */
+  static final class StepVerifierStepExpectNextCountZero<T> {
+    @BeforeTemplate
+    StepVerifier.Step<T> before(StepVerifier.Step<T> step) {
+      return step.expectNextCount(0L);
+    }
+
+    @AfterTemplate
+    StepVerifier.Step<T> after(StepVerifier.Step<T> step) {
+      return step;
+    }
+  }
```

Comment:
Sorry, I just noticed: This rule and `StepVerifierStepExpectNextEmpty` have the same `after` step. We should join those into one.
Usually, these rules are named by the resulting code, see other examples in this class. Since there is "no" resulting code here, I'd then name this one `StepVerifierStepIdentity`. :+1:

Reply:
@mohamedsamehsalah Let's also rename?